contentScript.js

```diff
@@ -29,16 +29,7 @@ if (center_prefs.length === 0) {
 }
 
 
-let booking_lower_lim = 1;
-try {
-  booking_lower_lim = parseInt(minavailability);
-} catch (e) {
-  booking_lower_lim = 1;
-} finally {
-  if (isNaN(booking_lower_lim)) {
-    booking_lower_lim = 1;
-  }
-}
+let booking_lower_lim = minavailability ? parseInt(minavailability) : 1;
```

Comment:
What if
```
minavailability = "abc";
```

Reply:
For such case, let booking_lower_lim = isNaN(parseInt(minavailability)) ? 1 :parseInt(minavailability);
By the way, there will no such case where minavailability will have String other than a Number. In my opinion, checking NaN for such cases is redundant & should be avoided.

Reply:
If someone, out of experimentation edits the local storage directly, the script would stop working.

Reply:
> For such case, let booking_lower_lim = isNaN(parseInt(minavailability)) ? 1 :parseInt(minavailability);
> By the way, there will no such case where minavailability will have String other than a Number. In my opinion, checking NaN for such cases is redundant & should be avoided.

Readability?

Reply:
I have changed a bit of code and made it readable. Also there was unnecessary catch which was removed as parseInt will never throw error. And thus no need of try and finally.